scripts/config/cspell/dictionaries/qiskit.txt

```diff
@@ -132,6 +132,7 @@ ipywidgets
 iqft
 iswap
 ISYM
+jobid
```

Comment:
Not sure this should be in the dictionary, I think we should either say " job ID" in a sentence, or use code stying (`jobid`) if referring to the method / variable.